AmplifyPlugins/Core/AWSPluginsCore/Model/Support/QueryPredicate+GraphQL.swift

```diff
@@ -47,11 +48,22 @@ public struct GraphQLFilterConverter {
 /// Extension to translate a `QueryPredicate` into a GraphQL query variables object
 extension QueryPredicate {
 
+    public func graphQLFilter(_ modelSchema: ModelSchema? = nil) -> GraphQLFilter {
+        if let operation = self as? QueryPredicateOperation {
+            return operation.graphQLFilter(modelSchema)
+        } else if let group = self as? QueryPredicateGroup {
+            return group.graphQLFilter(modelSchema)
+        }
+
+        preconditionFailure(
+            "Could not find QueryPredicateOperation or QueryPredicateGroup for \(String(describing: self))")
+    }
+
     public var graphQLFilter: GraphQLFilter {
         if let operation = self as? QueryPredicateOperation {
-            return operation.graphQLFilter
+            return operation.graphQLFilter(nil)
```

Comment:
nit: do we need to pass `nil` as argument here and on line 66? Looks like method argument has `nil` as default value

Reply:
Changed to `graphQLFilter()`